Documentation/devel/stage1-implementors-guide.md

```diff
@@ -118,3 +118,22 @@ Examples
     ]
 }
 ```
+
+Filesystem Layout Assumptions
+-----------------------------
+
+### opt/stage2
+
+As mentioned above, in rkt, the apps will be executed during stage2.
+rkt expects the directory path `opt/stage2` to exist under the stage1 rootfs.
```

Comment:
This is actually created by stage0:

* Overlay: https://github.com/coreos/rkt/blob/e04a470e01f53f99ff81f6f4be7cf4815eff5ea0/stage0/run.go#L556
* Non-overlay: https://github.com/coreos/rkt/blob/e04a470e01f53f99ff81f6f4be7cf4815eff5ea0/stage0/run.go#L525